feat: change gitpod layout (#1070)
tendermint/starport: .gitpod/readme.md

# Welcome to Starport ✨

> Starport is an easy to use CLI tool for creating sovereign blockchains. Blockchains created with Starport use Cosmos SDK, the world's most widely-used blockchain application framework.

Reviewer: ```suggestion
Starport is an easy-to-use CLI tool for creating sovereign blockchains. Blockchains created with Starport use Cosmos SDK, the world's most widely used blockchain application framework.
```

> In front of you is a browser-based development environment. At the bottom of the screen you can see a terminal window, it has `starport` binary pre-installed.

Reviewer: ```suggestion
In this browser-based development environment, the terminal window is in the lower part of the window. The `starport` binary is pre-installed and ready to use on the command line.
```

Reviewer: tricky to describe positional UI, per https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/lower-left-lower-right

Reviewer 2: That's really good!

## Quick start

> Create a new blockchain and start a node in development:

Reviewer: ```suggestion
To create a blockchain and start a node in development:
```

Reviewer: create and new are redundant (I know, we all do it!)

```
starport app github.com/user/hello

cd hello

starport serve
```

## Next steps

> 📺 **[Introduction](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: watch a quick introduction to Starport.

Reviewer: ```suggestion
📺 **[Introduction to Starport video](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: watch an introductory video to learn about Starport.
```

> 📺 **[Introduction to Starport](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: watch an introductory video to learn about Starport.

Reviewer: ```suggestion
📺 **[Introduction to Starport](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: Watch an introductory video to learn about Starport.
```

Reviewer: thanks for the explanation of this new and improved user experience in gitpod! Let's make this one last change. @fadeev

> 🧑‍🏫 **[Tutorials](https://tutorials.cosmos.network)**: learn by building a simple IBC-enabled module, nameservice or a decentralized exchange (DEX).

Reviewer: ```suggestion
🧑‍🏫 **[Tutorials](https://tutorials.cosmos.network)**: Learn by building a simple IBC-enabled module, nameservice, or a decentralized exchange (DEX).
```

> 📕 **[Starport Documentation](https://docs.starport.network)**: explore features of Starport.

Reviewer: ```suggestion
📕 **[Starport Documentation](https://docs.starport.network)**: Explore the features of Starport.
```

> 📚 [Cosmos SDK Documentation](https://docs.cosmos.network): learn more about the framework.

Reviewer: ```suggestion
📚 [Cosmos SDK Documentation](https://docs.cosmos.network): Learn about the framework for building application-specific blockchains.
```

> ⭐️ [Starport on Github](https://github.com/tendermint/starport): submit an issue or contribute to the source code.

Reviewer: ```suggestion
⭐️ [Starport on Github](https://github.com/tendermint/starport): Submit an issue or contribute to the source code.
```

## Starport features

> * Scaffold modules, messages, types with CRUD operations, IBC packets and more

Reviewer: ```suggestion
* Scaffold modules, messages, types with CRUD operations, IBC packets, and more
```

Reviewer: hi Denis, you know I'm passionate about the serial comma (aka the Harvard comma)

> * Start a blockchain node in development with hot-reloading

Reviewer: what exactly is hot-reloading? and why do we hyphenate it?

Reviewer 2: Technically it's "live reloading" (restarting a program on changes to the source code), and we don't need the hyphen, yes.

> * Start a blockchain node in development with hot-reloading

Reviewer 2: ```suggestion
* Start a blockchain node in development with live reloading
```

* Connect to other blockchains with a built-in IBC relayer
* Use automatically generated TypeScript/Vuex clients to interact with your blockchain

> * Vue.js web app template with a set of components and Vuex modules

Reviewer: ```suggestion
* Use the Vue.js web app template with a set of components and Vuex modules
```

Reviewer: I don't know is "Use the" is the right wording here, but let's start with an action verb (maybe "Build...?)

Reviewer 2: "Use" is ok.
